packages/breadcrumb/src/BreadcrumbContainer.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import PropTypes from 'prop-types';
+
+import useBreadcrumb, { IUseBreadCrumbReturnValue } from './useBreadcrumb';
```

Comment:
Minor but to keep it consistent I would not capitalise the C e.g. `IUseBreadcrumbReturnValue`